hashicorp/boundary-ui
# chore: 🤖 Update Cross OS

## Description

Update Cross OS

### How to test:

Run `prebuild:desktop` since is the one using `cross-os`.

Also run the desktop release CI to triple check everything still working:

Prove the Desktop release CI run and successful

![Screen Shot 2022-06-28 at 1 19 25 PM](https://user-images.githubusercontent.com/9775006/176277477-7a647ad9-4e27-48f9-9235-82cbfc9eeaaa.png)
![Screen Shot 2022-06-28 at 1 20 03 PM](https://user-images.githubusercontent.com/9775006/176277485-94d3028b-aa59-41fd-97d1-fe022414c23b.png)
